inclusive-design/idrc
# GH-540 and GH-515: reduce image filesize to improve performance and delete unused image files

## Description

Reduce files size on many images by resizing, reducing bit-depth, or changing formats. Also removed images that were present in the media directory but not used anywhere in the source code.

Addresses #540 and #515 .

## Steps to test

1. Run the server
2. Check to see that all images are being rendered (may need to clear browser cache)
